packages/compass-aggregations/src/components/pipeline-explain/explain-error.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  css,
+  spacing,
+  Subtitle,
+  ErrorSummary,
+  Button,
+} from '@mongodb-js/compass-components';
+
+type ExplainErrorProps = {
+  isNetworkError: boolean;
+  message: string;
+  onRetry: () => void;
+};
+
+const containerStyles = css({
+  display: 'flex',
+  gap: spacing[2],
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  marginTop: spacing[4],
+  marginBottom: spacing[4],
+});
+
+const textStyles = css({
+  textAlign: 'center',
+  marginTop: spacing[4],
+  marginBottom: spacing[4],
+});
+
+export const ExplainError: React.FunctionComponent<ExplainErrorProps> = ({
+  isNetworkError,
+  message,
+  onRetry,
+}) => {
+  if (!isNetworkError) {
+    return (
+      <ErrorSummary data-testid="pipeline-explain-error" errors={[message]} />
+    );
+  }
```

Comment:
I’m curious about this. We’re not giving network errors special treatment in other parts of Compass – where did we make the decision to add this? If we do want to handle network errors, it might make sense to think about how we can address them more generally, not just for explain execution (especially since users are unlikely to reach the point of being able to run explain on aggregations if there are connectivity issues).

Reply:
This came from the designs that Claudia shared with me. I'll ping her, see if can align the workflow and avoid this use case.

Reply:
Yeah, I mean, it's not like anything's bad about this, it just feels like we're doing a lot of work to handle a very unlikely error condition well